notice_and_comment/static/regulations/css/less/fonts.less

```diff
@@ -0,0 +1,29 @@
+/*
+ Fonts
+ ==========================================================================
+ Place all @font-face rules in fonts.less
+ */
+
+@font-face {
```

Comment:
Mostly to make sure I know what I'm doing in the future, is there a reason for separating out the different font-face rules into their own font-family names as opposed to rolling them all under 'Source Sans Pro'? 

Doesn't this make things fail less gracefully when a browser has to fall back to a non-web font?  Or does that get covered in the stuff from -site we are over-riding?